lapce-proxy/src/plugin/mod.rs

```diff
@@ -488,7 +489,16 @@ impl PluginCatalogRpcHandler {
                 start: position,
                 end: position,
             },
-            context: CodeActionContext::default(),
+            context: CodeActionContext {
+                diagnostics: diagnostics
+                    .into_iter()
+                    .filter(|x| {
```

Comment:
Maybe it's better to filter the diagnostics in the code actions request.

Reply:
Done